File: python/_etc/make_folder.py
import os

# 몇개의 폴더를 생성?
chap_num = int(input("몇 개의 폴더를 생성할까요? > "))

dir_name = "chap00"

# 폴더 만들기
for i in range(1, chap_num+1):
    try:
        dir_name = f"chap{i:02}"  # chap01...
        os.mkdir(dir_name)
    except:
        print("해당폴더가 존재합니다.")

print("폴더 생성을 완료하였습니다.")

File: javascript/_etc/makeFolder.js
const fs = require('fs');

const makeFolder = (dirname) => {
  if (!fs.existsSync(dirname)) {
    fs.mkdirSync(dirname);
  }
};

// 몇개의 폴더를 생성?
const chapNum = 12

let dirname = 'chap00';

for (let i = 1; i <= chapNum; i++) {
  if (i < 10)
    dirname = `./chap0${i}`;
  else
    dirname = `./chap${i}`;
  makeFolder(dirname);
}

console.log('폴더 생성을 완료하였습니다.');
